pkgs/build-support/build-bazel-package/default.nix

```diff
@@ -50,13 +53,17 @@ let
     bazelFlags = bazelFlags;
     bazelBuildFlags = bazelBuildFlags;
     bazelTestFlags = bazelTestFlags;
+    bazelRunFlags = bazelRunFlags;
+    runTargetFlags = runTargetFlags;
     bazelFetchFlags = bazelFetchFlags;
+    bazelTargets = bazelTargets;
     bazelTestTargets = bazelTestTargets;
+    bazelRunTarget = bazelRunTarget;
     dontAddBazelOpts = dontAddBazelOpts;
```

Comment:
Why not do inherit here?

Reply:
in this case I was simply following the source as it's currently written, aiming to minimize git blame noise